src/modelgauge/tests/safe.py

```diff
@@ -146,18 +135,13 @@ def make_test_items(self, dependency_helper: DependencyHelper) -> List[TestItem]
                     )
         return test_items
 
-    def get_annotators(self) -> Mapping[str, Annotator]:
-        return {
-            "llama_guard": LlamaGuardAnnotator(
-                "llama_guard",
-                self.together_api_key,
-                llama_guard_version=self.llama_guard_version,
-            )
-        }
+    @classmethod
+    def get_annotators(cls) -> List[str]:
```

Comment:
It would be cool to be able to use a class property for this, rather than a `get_x` function, but it turns out it's pretty nasty to implement. So we're better off with `get_annotators`.  

https://stackoverflow.com/questions/5189699/how-to-make-a-class-property

Reply:
Yeah that would definitely a better option if it was simpler